recipes/libraw/all/conanfile.py

```diff
@@ -60,14 +62,20 @@ def requirements(self):
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
             check_min_cppstd(self, self._min_cppstd)
+            if Version(self.version) <= "0.19.5":
+                if conan_version.major >= 2:
+                    check_max_cppstd = getattr(sys.modules['conan.tools.build'], 'check_max_cppstd')
+                    check_max_cppstd(self, 11)
 
     def source(self):
-       get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
     def generate(self):
         tc = CMakeToolchain(self)
         tc.variables["RAW_LIB_VERSION_STRING"] = self.version
         tc.variables["LIBRAW_SRC_DIR"] = self.source_folder.replace("\\", "/")
+        if conan_version.major == 1:
+            tc.variables["CMAKE_CXX_STANDARD"] = 11
```

Comment:
```suggestion
        if conan_version.major == 1:
            tc.variables["CMAKE_CXX_STANDARD"] = self.settings.get_safe("compiler.cppstd", 11)
```

In case a version does not support C++17/20, for instances, let's not try workaround, but just avoid building with check_max_cppstd, like: https://github.com/conan-io/conan-center-index/blob/master/recipes/ccfits/all/conanfile.py#L52